site/styles/_share-modal.scss

```diff
@@ -58,16 +58,17 @@
     }
   }
 
-  [type="button"] { // for testers on their ipod touch
-    -webkit-appearance: none;
+  [type='button'] {
+    // for testers on their ipod touch
+    appearance: none;
```

Comment:
Double up?

Reply:
I removed the double. Hopefully it works without the webkit specific styles. Otherwise we will have to change it back every time the linter is run.